### PR TITLE
upgraded to Play 2.7.0-RC3

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +49,9 @@ The Play Slick plugin supports several different versions of Play and Slick.
 | 0.6.1          | 2.2.x        | 2.0.x         | 2.10.x        |
 | 0.5.1          | 2.2.x        | 1.0.x         | 2.10.x        |
 
+
+> * Release Candidate: these releases are not stable and should not used in be production.
```
Comment (ignasi35) [inline README.md]:
```suggestion
> * Release Candidate: these releases are not stable and should not be used in production.
```

**Comment by renatocaval** [comment]
> Also, build is broken on (legit?) test failures.

I don't know why the build was failing.

**Comment by dwijnand** [comment]
```
[info] DefaultSlickApiSpec
[info] 
[info] DefaultSlickApi should
[info]   + check the assumption that the ApplicationLifecycle isn't null
[info]   + check the assumption that the ApplicationLifecycle is a singleton
[info]   + check the assumption that ApplicationLifecycle is binded to a DefaultApplicationLifecycle
[error]   x check that no stop hook is registered in the ApplicationLifecycle at application start
[error]    'Array(play.mvc.FileMimeTypesProvider$$Lambda$581/1478547096@1721befb)' doesn't have size 0 but size 1 (DefaultSlickApiSpec.scala:43)
[info] 
[error]   x check that a stop hook is registered in the ApplicationLifecycle when a database is created
[error]    'Array(play.api.db.slick.DefaultSlickApi$DatabaseConfigFactory$$Lambda$871/460127738@2c69675c, play.mvc.FileMimeTypesProvider$$Lambda$581/1478547096@1721befb)' doesn't have size 1 but size 2 (DefaultSlickApiSpec.scala:51)
```

Have a look at @marcospereira's https://github.com/playframework/play-slick/commit/55d9db21d92256b6dc94862318d3d0298257ce80#diff-c52b7eb4143c2019aebe42c1e286617a.

**Comment by renatocaval** [comment]
@dwijnand indeed, we know how to fix it, but I would like to understand why that changed between 2.7.0-M4 and 2.7.0-RC1

**Comment by dwijnand** [comment]
I agree it's worth investigating first.  I thought that change might provide some initial insight 🙂

File: src/core/src/test/scala/play/api/db/slick/DefaultSlickApiSpec.scala [hunk 2]
```diff
@@ -24,31 +24,36 @@ class DefaultSlickApiSpec extends Specification with Mockito { self =>
...
     }
+
     "check the assumption that ApplicationLifecycle is binded to a DefaultApplicationLifecycle" in {
       val lifecycle = injector.instanceOf[ApplicationLifecycle]
       lifecycle must beAnInstanceOf[DefaultApplicationLifecycle]
     }
-    "check that no stop hook is registered in the ApplicationLifecycle at application start" in {
-      val lifecycle = injector.instanceOf[ApplicationLifecycle].asInstanceOf[DefaultApplicationLifecycle]
-      hooks(lifecycle) must have size (0)
-    }
```
Comment (renatocaval) [inline src/core/src/test/scala/play/api/db/slick/DefaultSlickApiSpec.scala]: this test is not adding any value to play-slick. Why would we test that there is not hook added by Play?

What we need to confirm is that `SlickApi` is adding its own hook. That's all.
See next test for more context.

**Pull request merged**